src/codewhisperer/activation.ts

```diff
@@ -244,65 +247,15 @@ export async function activate(context: ExtContext): Promise<void> {
                     return
                 }
             }
-
+            await globals.context.globalState.update(CodeWhispererConstants.accessToken, undefined)
+            await globals.context.globalState.update(CodeWhispererConstants.accessTokenExpriedKey, true)
             await vscode.commands.executeCommand('aws.codeWhisperer.refreshRootNode')
-            const t = new Date()
-
-            if (t <= CodeWhispererConstants.accessTokenCutOffDate) {
-                maybeShowTokenMigrationWarning()
-            } else {
-                await globals.context.globalState.update(CodeWhispererConstants.accessToken, undefined)
-                await globals.context.globalState.update(CodeWhispererConstants.accessTokenExpriedKey, true)
-                await vscode.commands.executeCommand('aws.codeWhisperer.refreshRootNode')
-                maybeShowTokenMigrationError()
-            }
+            maybeShowTokenMigrationError()
         } else if (accessTokenExpired) {
             maybeShowTokenMigrationError()
         }
     }
 
-    function maybeShowTokenMigrationWarning() {
```

Comment:
some cleanup since we don't need this code path anymore

Reply:
nice